writing_checks/README.md

```diff
@@ -468,6 +468,38 @@ whereas if it run on a system with ALPS it will be translated to
 ddt --offline aprun ...
 ```
 
+## Custom Launchers
+
+ReFrame permits the simple addition of custom scheduler launchers. A launcher that invokes `mpirun` can for example be added as follows:
+
+```python
+class MpirunLauncher(JobLauncher):
+    @property
+    def executable(self):
+        return 'mpirun'
+
+    @property
+    def fixed_options(self):
+        return [ '-np %s' % self.job.num_tasks ]
+```
+It will be translated to `mpirun -np ...`. While the definition of the property `executable` is obviously mandatory, the definition of `fixed_options` is optional; it defaults to no options. Note that the launcher class may use the information contained in `self.job`.
```

Comment:
Here the reference to `self.job` is a bit misleading. You should somehow explain that a launcher is associated with a job descriptor, from which it can extract the job submission information.